initial.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
```

Comment:
Nice job, my only suggestion is that you add a comment with a description of the program at the top.